python/objToJSON.c

```diff
@@ -259,46 +259,58 @@ static int Dict_iterNext(JSOBJ obj, JSONTypeContext *tc)
     return 0;
   }
 
-  if (!(GET_TC(tc)->itemValue = PyObject_GetItem(GET_TC(tc)->dictObj, GET_TC(tc)->itemName)))
-  {
+  if (GET_TC(tc)->itemValue) {
```

Comment:
Oh, I hadn't seen this and fixed it differently in https://github.com/esnme/ultrajson/pull/353.